### PR TITLE
Update Version to 4.2.5.1.0

### PR DESCRIPTION
**Pull request by helium-service**
Adapter version '4.2.5.1.0', partner version '~> 2.5.1'

### CODE REVIEW

**Comment by bwised** [comment]
Blocked by https://github.com/ChartBoost/chartboost-mediation-ios-adapter-bidmachine/pull/7.  Branched from `release/4.2.5.0.0`.

**Comment by daniel-barros** [comment]
@bwised can we close this PR and remove the release branch? As discussed, the 2.5.0 update should suffice.

**Comment by daniel-barros** [comment]
> @bwised can we close this PR and remove the release branch? As discussed, the 2.5.0 update should suffice.

oh wait are we actually releasing a 2.5.1 version?

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,12 @@
 Note the first digit of every adapter version corresponds to the major version of the Chartboost Mediation SDK compatible with that adapter. 
 Adapters are compatible with any Chartboost Mediation SDK version within that major version.
 
+### 4.2.5.1.0
+- This version of the adapter has been certified with BidMachine 2.5.1.
+
+### 4.2.5.0.0
+- This version of the adapter has been certified with BidMachine 2.5.0.
+
```
Comment (kendallrogers) [inline CHANGELOG.md]: If we're not releasing 2.5.0, we can probably just remove this entry.

**Comment by bwised** [comment]
Last message from @pleasesavemycat  that I saw was to move forward with the ticket assigned to me.  If that's no longer true, I can close everything out.

File: CHANGELOG.md (hunk 1 above)
Comment (daniel-barros) [reply CHANGELOG.md]: we just released it

**Comment by daniel-barros** [comment]
> Last message from @pleasesavemycat that I saw was to move forward with the ticket assigned to me. If that's no longer true, I can close everything out.

maybe I misunderstood sorry, I asked Kelly in the channel to clarify.
this update should be unnecessary, but maybe we want to release it to reduce the chances that some pub uses the bidmachine 2.5.0 version if they haven't done a pod update

**Comment by daniel-barros** [comment]
> > Last message from @pleasesavemycat that I saw was to move forward with the ticket assigned to me. If that's no longer true, I can close everything out.
> 
> maybe I misunderstood sorry, I asked Kelly in the channel to clarify. this update should be unnecessary, but maybe we want to release it to reduce the chances that some pub uses the bidmachine 2.5.0 version if they haven't done a pod update

all right Kelly confirmed we are moving forward with the update, sorry for the delay

**Review by daniel-barros — APPROVED** [review]

**Pull request merged**